xla/service/gpu/cudnn_fused_mha_transpose_fusion.cc

```diff
@@ -271,8 +272,8 @@ StatusOr<bool> FuseArgPrologueTransposeWithcuDNNFMHA(
     }
   }
 
-  fmha->clear_backend_config();
-  TF_RETURN_IF_ERROR(fmha->set_backend_config(new_fmha_config));
+  *gpu_config.mutable_cudnn_fmha_backend_config() = new_fmha_config;
```

Comment:
ditto for this and other occurrences as I commented in xla/service/gpu/conv_algorithm_picker.cc